src/operator/l2_normalization-inl.h

```diff
@@ -314,6 +309,11 @@ class L2NormalizationProp : public OperatorProperty {
     return {{out_grad[l2_normalization::kOut], in_grad[l2_normalization::kData]}};
   }
 
+  std::vector<ResourceRequest> ForwardResource(
+      const std::vector<TShape> &in_shape) const override {
+    return{ ResourceRequest::kTempSpace };
+  }
+
```

Comment:
overall looks good, can you add a test for the failing test case that you mentioned in the issue.

Reply:
The main problem is that we don't know the cause of this problem. It is difficult to build a test case. @anirudh2290